packages/cogify/src/cogify/gdal/gdal.command.ts

```diff
@@ -2,18 +2,22 @@ import { Rgba } from '@basemaps/config';
 import { Epsg, EpsgCode, TileMatrixSets } from '@basemaps/geo';
 import { urlToString } from '@basemaps/shared';
 
-import { Presets } from '../preset.js';
+import { Presets } from '../../preset.js';
+import { CogifyCreationOptions } from '../stac.js';
 import { GdalCommand } from './gdal.runner.js';
-import { CogifyCreationOptions } from './stac.js';
 
 const isPowerOfTwo = (x: number): boolean => (x & (x - 1)) === 0;
+const DEFAULT_TRIM_PIXEL_RIGHT = 1.7; // 1.7 pixels to trim from the right side of the topo raster imagery
 
-export function gdalBuildVrt(targetVrt: URL, source: URL[]): GdalCommand {
+export function gdalBuildVrt(targetVrt: URL, source: URL[], addalpha?: boolean): GdalCommand {
   if (source.length === 0) throw new Error('No source files given for :' + targetVrt.href);
   return {
     output: targetVrt,
     command: 'gdalbuildvrt',
-    args: [urlToString(targetVrt), ...source.map(urlToString)],
+    args: [addalpha ? ['-addalpha'] : undefined, urlToString(targetVrt), ...source.map(urlToString)]
+      .filter((f) => f != null)
```

Comment:
I wonder if `flatMap` would work here

```
.flatMap(f => f != null ? String(f): [])


[['foo'], undefined, ['bar'], [undefined, 'baz']].flatMap(f => f != null ? String(f): [])
[ 'foo', 'bar', ',baz' ]
```

Reply:
This only works with one value in the nested array, and not working with others with multiple values like `[ ['-of', 'vrt'], ]`